Makefile

```diff
@@ -4,13 +4,11 @@ PREFIX ?= /
 
 .PHONY: install
 install:
-	install -D -m0644 src/overdue.awk $(PREFIX)/usr/share/overdue/overdue.awk
 	install -D -m0644 src/overdue-libraries-upgrade.hook \
 		$(PREFIX)/usr/share/libalpm/hooks/overdue-libraries-upgrade.hook
-	install -D -m0755 src/overdue.sh  $(PREFIX)/usr/bin/overdue
```

Comment:
One thing though - because the list is not coming from pacman, this isn't telling us which library files have changed, so it reports all running systemd services that reference any library file, not just the ones that are stale. I'll have to think about that.